src/components/cycle-section/cycle-section/CycleSection.js

```diff
@@ -46,7 +53,13 @@ const CycleSection = ({ cycleId }) => {
             <ChallengeCard cycle={cycle} />
           </div>
           <div className={styles.bottomContainer}>
-            <LearningUnitsCard cycle={cycle} learningUnits={learningUnits} />
+            <LearningUnitsSection
+              learningUnits={learningUnits}
+              successions={successions}
+              handleLearningUnitClick={(learningUnitId, isCompleted) =>
+                handleLearningUnitRedirection(learningUnitId, isCompleted)
```

Comment:
The naming convention seems off: I think it should be called handleLearningUnitClick from the beginning, since it handles clicks, not redirections.